assets/custom.css

```diff
@@ -0,0 +1,32 @@
+body {
+    font-size: 0.95em;
+}
+
+h1, h2, h3, h4, h5 {
+    text-align: left !important;
+}
+
+.remark-slide-content {
+    background: white !important;
+    text-align: left;
+}
+
+/** BEGIN First slide. ********************************************************/
+
+.remark-slide-container:first-child .remark-slide-content:after {
+    content: '';
+    background: url("../figures/logo_ulg.png") no-repeat;
+    background-size: 100% 100%;
+    width: 150px;
+    height: 73px;
+    position: absolute;
+    bottom: 2em;
+    left: 3.3em;
+    display: inline-block;
+}
+
+.remark-slide-container:first-child .remark-slide-content {
+    vertical-align: middle;
+}
+
+/** END First slide. **********************************************************/
```

Comment:
Can we move all this into `style.css`? (which is already my custom css styles wrt to remark's)

Reply:
Didn't realize that. Sorry, will fix.